algorithms/ar-cochprb/javascript/coin_change_problem.js

```diff
@@ -0,0 +1,21 @@
+
+function waysofChange(s, numOfCoins, n) {
+    
+    if(n === 0) {
+        return 1; 
+    }
+    
+    if(n < 0){
+        return 0; 
+    }
+    
+    if(numOfCoins < 0 && n > 0){
+        return 0; 
+    }
+
+    return waysofChange(s, numOfCoins, n - s[numOfCoins]) + waysofChange(s, numOfCoins - 1, n);
+}
+
+var s = [1, 2, 3];
+var n = 4;
+console.log(waysofChange(s, s.length - 1, n));
```

Comment:
Codacy found an issue: [Unexpected console statement.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6356108)

Reply:
any idea how to fix this?